Cerberus.Dashboard.Domain/Models/Application.cs

```diff
@@ -16,10 +16,19 @@ public class Application : Entity
         public string Ethnicity { get; set; }
         public string ApplicationDate { get; set; }
         public bool IsDisable { get; set; }
+        public string ApplicationStage { get; set; }
+        public string PhysicalAddressLine1 { get; set; }
+        public string PhysicalAddressLine2 { get; set; }
+        public string PhysicalAddressLine3 { get; set; }
+        public string PhysicalAddressCity { get; set; }
+        public string PhysicalAddressPostalCode { get; set; }
+        public string PostalAddressLine1 { get; set; }
+        public string PostalAddressLine2 { get; set; }
+        public string PostalAddressLine3 { get; set; }
+        public string PostalAddressCity { get; set; }
```

Comment:
Since we are repeating these fields why Dont we have an Address Entity separated by AddressType(Physical or Postal and  Work)